remotespark/livyclientlib/livysession.py

```diff
@@ -50,9 +51,9 @@ def __init__(self, http_client, properties, ipython_display,
         self._status_sleep_seconds = status_sleep_seconds
         self._statement_sleep_seconds = statement_sleep_seconds
         self._create_sql_context_timeout_seconds = create_sql_context_timeout_seconds
+        self.session_guid = uuid.uuid4()
```

Comment:
We should talk about GUIDs for events. There's a bunch of operations that will need GUIDs. We shouldn't pollute all of our methods...

I believe the GUID should be handled by `SparkEvents` behind the covers. Let's talk about it tomorrow.